### PR TITLE
chore: update ingress-nginx image

### PR DESCRIPTION
**Pull request by coadler**
[ch12199]

### CODE REVIEW

**Review by deansheather — APPROVED** [review]

File: templates/ingress.yaml [hunk 1]
```diff
@@ -259,8 +259,8 @@ spec:
                 - ALL
               add:
                 - NET_BIND_SERVICE
-            # www-data -> 33
-            runAsUser: 33
+            # www-data -> 101
+            runAsUser: 101
```
Comment (jawnsy) [inline templates/ingress.yaml:263]: we don't store any files right? this won't affect any sort of filesystem permission things?

**Review by jawnsy — APPROVED** [review]

File: templates/ingress.yaml (hunk 1 above)
Comment (coadler) [reply templates/ingress.yaml:263]: they're all via configmaps it seems, didn't notice anything break in my testing

Comment (deansheather) [reply templates/ingress.yaml:263]: no volumes, and i verified that the user is correct by pulling the image manually

Comment (jawnsy) [reply templates/ingress.yaml:263]: cool, sounds good, thanks for checking :+1:

**Pull request merged**